src/shared/icons/index.tsx

```diff
@@ -10,7 +10,7 @@ export { HtmlIcon } from './html-icon';
 export { InstagramIcon } from './instagram';
 export { JavascriptIcon } from './javascript-icon';
 export { JetBrainsLogo } from './jetbrains';
-export { LinkedInIcon } from './linkedIn';
+export { LinkedInIcon } from './linked-in';
```

Comment:
```suggestion
export { LinkedInIcon } from './linkedin';
```
because it's 1 word (like "vkontakte" - you don't write "v-kontakte")